settings.gradle

```diff
@@ -1,25 +1,7 @@
 rootProject.name = 'protobuf-gradle-plugin'
 
-// Include test projects only when we intend to run the tests.
-// We have to exclude them when we intend to build the plugin, because the test
-// projects apply the plugin at evaluation time. At evaluation time the plugin
-// has not been compiled yet.
 if (gradle.startParameter.taskNames.intersect(['install', 'uploadArchives', 'publishPlugins'])) {
```

Comment:
This was needed because the plugin needed to be installed from a separate command prior to being tested. You can remove it now.